app/scripts/modules/core/src/notification/selector/types/githubstatus/GithubNotificationType.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
```

Comment:
We typically add a newline after third-party imports per this (hard to find) conventions doc: https://github.com/spinnaker/deck/wiki/Conventions#imports
```suggestion
import * as React from 'react';

```

Reply:
i've updated the PR and added the new line.